lib/consts.js

```diff
@@ -5,6 +5,7 @@ dotenv.config();
 export const AI_MODEL = "gpt-4o-mini";
 export const APIFY_TOKEN = process.env.APIFY_TOKEN;
 export const MAX_TWEETS = 1000;
+export const SPOTIFY_CLIENT_ID = "f101e88980ad4f32a04cba38d4cff851";
```

Comment:
Is this a private variable associated with our spotify account?
- If yes, please remove from `public` repo.